applications/RomApplication/python_scripts/hrom_training_utility.py

```diff
@@ -97,7 +98,7 @@ def CreateHRomModelParts(self):
         if  self.hrom_output_format == "numpy":
             hrom_info = KratosMultiphysics.Parameters(json.JSONEncoder().encode(self.__CreateDictionaryWithRomElementsAndWeights()))
         elif self.hrom_output_format == "json":
-            with open('RomParameters.json','r') as f:
+            with open(self.rom_basis_output_name + ".json",'r') as f:
```

Comment:
```suggestion
            with open("{self.rom_basis_output_name}.json",'r') as f:
```
Since we dropped Python2 support we can use f-strings.

Reply:
Changing to:
```
f"{self.rom_basis_output_name}.json"
```